CONTRIBUTING.md

```diff
@@ -0,0 +1,188 @@
+<div align="left">
+<img src="https://raw.githubusercontent.com/wetransform-os/ProMCDA/release/logo/ProMCDA_logo.png">
+<div>
+
+## Contributing to Probabilistic Multi Criteria Decision Analysis, ```ProMCDA```.
+
+### Table of Contents
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+- [Getting started](#getting-started)
+- [Issue tracking](#issue-tracking)
+- [Making changes](#making-changes)
+- [Code style](#code-style)
+- [Testing](#testing)
+- [Project origins and author affiliations](#affiliation)
+- [Mantainers](#mantainers)
+
+### Introduction
+Welcome to ```ProMCDA```! We're thrilled that you're interested in contributing to our project. Whether you're fixing 
+a bug, implementing a new feature, or improving documentation, your contributions help make ```ProMCDA``` better for 
+everyone.
+
+Before you get started, please take a moment to review the following guidelines to ensure that your contributions are 
+effective and in line with our project's standards. If you have any questions or need assistance, don't hesitate to 
+reach out to the maintainers.
+
+### Prerequisites
+Before contributing to ```ProMCDA```, ensure that you have the following prerequisites installed:
+
+- Python (version Python 3.9)
+- Pip (version 23.2.1)
+- Other project-specific dependencies listed in the `requirements.txt` file
+
+You can install the Python dependencies listed in the `requirements.txt` file using pip. For example:
+
+```bash
+pip install -r requirements.txt
+```
+
+### Getting Started
+First, you'll need to fork the project repository and clone it to your local machine. Go to the 
+[GitHub repository of the project] (https://github.com/wetransform-os/ProMCDA) and click the "Fork" button 
```

Comment:
I believe a space in `[GitHub repository of the project] (https://github.com/wetransform-os/ProMCDA)` makes the formatting incorrect. In the Readme.md file this will no longer be a hyperlink text but a text withing `[]` and a separate link.